syntax = "proto3";
package workflow;

import "parameter.proto";

message IntentInput {
  Parameter parameter = 1;
}